epam_tr/src/main/java/com/epam/tr/security/request/RequestProcessor.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.tr.security.request;
```

Comment:
I think it make sense to move this class to security package if there is only 1 file in request package.